src/Model/GServer.php

```diff
@@ -506,7 +503,7 @@ public static function cleanURL(string $dirtyUrl): string
 	 *
 	 * @param UriInterface $dirtyUri
 	 *
-	 * @return UriInterface cleaned URI
+	 * @return string cleaned URI
 	 * @throws Exception
 	 */
 	public static function cleanUri(UriInterface $dirtyUri): string
```

Comment:
In this case, the return type was wrong.
```suggestion
	 * @return UriInterface cleaned URI
	 * @throws Exception
	 */
	public static function cleanUri(UriInterface $dirtyUri): UriInterface 
```

Reply:
Done.